connman/include/setting.h

```diff
@@ -67,6 +67,7 @@ extern "C" {
 #define CONF_ENABLE_LOGIN_MANAGER            "EnableLoginManager"
 #define CONF_LOCALTIME                       "Localtime"
 #define CONF_REGDOM_FOLLOWS_TIMEZONE         "RegdomFollowsTimezone"
+#define CONF_DEFAULT_MDNS_CONFIGURATION      "DefaultmDNSConfiguration"
```

Comment:
Not sure if we'd need two options:
 1. A system wide setting overriding _all_ Ethernet/WLAN service mDNS
 2. Default setting for the new services, which is much more lighter option. 
 
However, these might then conflict, global being off but new getting enabled, or vice versa. But that use case might be possible, like with ours, that do not touch old ones but keep the setting on new ones. We can live with this current approach for now and add more functionality later.

Reply:
We live with this for now.